adapters/http-base/src/main/java/org/eclipse/hono/adapter/http/AbstractVertxBasedHttpProtocolAdapter.java

```diff
@@ -381,7 +383,10 @@ protected HttpServerOptions getHttpServerOptions() {
     protected HttpServerOptions getInsecureHttpServerOptions() {
 
         final HttpServerOptions options = new HttpServerOptions();
-        options.setHost(getConfig().getInsecurePortBindAddress()).setPort(getConfig().getInsecurePort(getInsecurePortDefaultValue())).setMaxChunkSize(4096);
+        options.setHost(getConfig().getInsecurePortBindAddress())
+               .setPort(getConfig().getInsecurePort(getInsecurePortDefaultValue()))
+               .setMaxChunkSize(4096)
+               .setIdleTimeout(getConfig().getIdleTimeout());
```

Comment:
The changes in this class refer to the HTTP adapter, so the documentation should be adapted as well, i.e. adding a row in the properties table in `http-adapter-config.md`.